src/data/TrackedEntityD2Repository.ts

```diff
@@ -0,0 +1,119 @@
+import _ from "lodash";
+import { D2Api } from "types/d2-api";
+import { Async } from "domain/entities/Async";
+
+import { getInChunks } from "./dhis2-utils";
+import { Stats } from "domain/entities/Stats";
+import {
+    TrackedEntityFilterParams,
+    TrackedEntityRepository,
+} from "domain/repositories/TrackedEntityRepository";
+import { TrackedEntity } from "domain/entities/TrackedEntity";
+import { D2TrackedEntity, ProgramsD2Repository } from "./ProgramsD2Repository";
+
+const initialStats = {
```

Comment:
`const initialStats: Stats`. Pros: 1) This way you get validation here (and not in usage), 2) On refactor (for example a field renaming), TS will know that you are here defining a Stat and act accodingly.